### DISCUSSION TITLE
《饿殍：明末千里行》：乱世中的羁绊 - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by giscus** [Muska-Ami/muska-ami-blog#76] [General]
# 《饿殍：明末千里行》：乱世中的羁绊 - 夏沫花火zzz🌙 (Muska_Ami)のLife

玩这款游戏其实只是偶然在 Steam 探索队列中看到的，因为发现有 Hanser 参与了配音，于是决定去玩玩。目前已经全结局全成就通关了，最后决定要浅谈一下该作品。 游戏 Steam 链接：https://store.steampowered.com/app/2593370  这应该是我第一个完整推完的 GalGame 作品吧。老实说，游戏给我留下的印象还是很深的。本文尽量不剧透，所以也就不深入谈

https://blog.1l1.icu/2024/06/24/e-piao-ming-mo-qian-li-xing-luan-shi-zhong-de-ji-ban/

**Comment by shiftz300** [comment]
人越简单
人就越复杂
食物也是这样的
game也是这样的
悲
